### DISCUSSION TITLE
Enlisted options?

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#45] [Q&A] [answered]
I know the operations division book has enlisted security, but where do the other enlisted track options come from? I vaguely recall seeing them, but I don't remember where.

Edit: Never mind, the core book says they're just the same as academy, except ranks options differ and command can't be taken. I think research "internship" and university are items I still don't know the source of.

**Comment by bcholmes** [comment, marked as answer]
University and Internship come from the Science Division sourcebook, pg. 34
